sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -177,9 +180,9 @@ def from_connection_string(cls, conn_str, consumer_group, **kwargs):
          information. The failed internal partition consumer will be closed (`on_partition_close` will be called
          if provided) and new internal partition consumer will be created (`on_partition_initialize` will be called if
          provided) to resume receiving.
-        :keyword float idle_timeout: Timeout in seconds after which the underlying connection will close
-         if there is no further activity. By default the value is None, meaning that the service determines when to
-         close an idle connection.
+        :keyword float idle_timeout: Timeout, in seconds, after which this client will close the underlying connection
+         if there is no furthur activity. By default the value is None, meaning that this client doesn't close but the
+         service determines when to close an idle connection.
```

Comment:
Same comment as above.

Reply:
Also applies to `producer_client.py` and async versions

Reply:
Updated to all consumer client and producer client constructors and from_connection_string